modules/tensor_mechanics/test/tests/creep_tangent_operator/tests

```diff
@@ -0,0 +1,50 @@
+[Tests]
+  [./ten]
+    type = 'CSVDiff'
+    input = 'creep.i'
+    csvdiff = 'creep_out.csv'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_ten"'
+  [../]
+  [./zero]
+    type = 'CSVDiff'
+    input = 'creep.i'
+    csvdiff = 'creep_out.csv'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_ten creep_zero"'
+    prereq = ten
+  [../]
+  [./sum]
+    type = 'CSVDiff'
+    input = 'creep.i'
+    csvdiff = 'creep_out.csv'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_nine creep_one"'
+    prereq = zero
+  [../]
+  [./cycle]
+    type = 'CSVDiff'
+    input = 'creep.i'
+    csvdiff = 'creep_out.csv'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_ten creep_ten2" Materials/elastic_strain/cycle_models=true'
+    prereq = sum
+  [../]
+
```

Comment:
It's a separate script from MOOSE so it _attempts_ to suppress all output, which is as good as it can since we don't have a master "turn everything off" flag. It should be fine.

Reply:
Ok, I'm good with this then.

Reply:
FYI, `analyzejacobian.py` writes out files that are prefixed with the input file name. So these AnalyzeJacobian can potentially interfere with each other.
I added prereqs in #11432